### PR TITLE
Add integration test using connetion string as auth method for App Configuration

### PR DESCRIPTION
**Pull request by ivywei0125**
Add integration test using connetion string as auth method for App Configuration.

### CODE REVIEW

File: sdk/spring/spring-cloud-azure-integration-test-appconfiguration-config/src/test/java/com/azure/spring/cloud/config/AppConfiguration.java [hunk 1]
```diff
@@ -9,13 +12,16 @@
 @Configuration
 public class AppConfiguration {
 
+    @Value("${authMethod}")
+    private String authMethod;
```
Comment (mrm9084) [inline sdk/spring/spring-cloud-azure-integration-test-appconfiguration-config/src/test/java/com/azure/spring/cloud/config/AppConfiguration.java:16]: Can you add a few comments here and a few other places. We don't touch this often. I wouldn't want to forget why we did this.

**Review by mrm9084 — CHANGES_REQUESTED** [review]

**Review by mrm9084 — APPROVED** [review]

**Comment by ivywei0125** [comment]
[like]  Yu Wei reacted to your message:
________________________________

**Pull request merged**